lib/improver/tests/acceptance/test_spot_extract.py

```diff
@@ -206,8 +206,8 @@ def test_percentile_deterministic(tmp_path):
     diag_path = kgo_dir / "inputs/ukvx_temperature.nc"
     kgo_path = kgo_dir / "outputs/nearest_uk_temperatures.nc"
     output_path = tmp_path / "output.nc"
-    args = [neighbour_path, diag_path, output_path,
-            "--extract_percentiles", "50"]
+    args = [neighbour_path, diag_path, "--output", output_path,
+            "--extract-percentiles", "50"]
     with pytest.warns(None):
```

Comment:
When looking through these tests quickly I noticed this one doesn't check the text of the warning and the rest of them do. Should we fix this?